test.js

```diff
@@ -5,3 +5,7 @@
 
 console.log('Hello other stuff');
 
+if(foo === "bar") {
+  console.log('stuff')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/DevK8scodacytestuser/codacy-test/pullRequest?prid=7692494)
```suggestion
  console.log("stuff")
```